### PR TITLE
copr: add `processed_versions_size` to ScanDetailV2

### PR DESCRIPTION
**Pull request by zhongzc**
Signed-off-by: Zhenchi <zhongzc_arch@outlook.com>

### CODE REVIEW

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -815,6 +815,12 @@ message ScanDetailV2 {
 
     // Total number of bytes from block reads.
     uint64 rocksdb_block_read_byte = 7;
+
+    // Total number of keys from the scan executor (i.e. TableScan or IndexScan) reads.
+    uint64 scan_executor_read_keys = 8;
```
Comment (breeswish) [inline proto/kvrpcpb.proto:820]: how about non executors (like KvScan)?

File: proto/kvrpcpb.proto [hunk 2]
```diff
@@ -815,6 +815,12 @@ message ScanDetailV2 {
 
     // Total number of bytes from block reads.
     uint64 rocksdb_block_read_byte = 7;
+
+    // Total number of keys from the scan executor (i.e. TableScan or IndexScan) reads.
```
Comment (breeswish) [inline proto/kvrpcpb.proto:819]: BTW this value may be exactly the same as `processed_versions` (please confirm).

Comment (zhongzc) [reply proto/kvrpcpb.proto:819]: ur right

**Comment by zhongzc** [comment]
Updated. cc @breeswish

**Review by breeswish — APPROVED** [review]

**Review by jyz0309 — APPROVED** [review]

File: proto/kvrpcpb.proto (hunk 1 above)
Comment (zhongzc) [reply proto/kvrpcpb.proto]: Found that `ScanResponse` doesn't have statistic fields,

https://github.com/pingcap/kvproto/blob/706fcaf286c8dd07ef59349c089f53289a32ce4c/proto/kvrpcpb.proto#L64-L72

is there a plan to add some?

**Pull request merged**